### PR TITLE
About page 2

### PR DESCRIPTION
**Pull request by parkhyunjoon**
Add About page default Container & Component setting
Replace .js to .jsx & Replace 4 indent to 2 indent

### CODE REVIEW

**Comment by woowong** [comment]
🦉

File: app/javascripts/about/emailContainer/emailContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    const emailInput = this.emailInput.value;
+    console.log(emailInput);
+    // e-mail validation by regular expression
+    const reg = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    if (!reg.test(emailInput)) {
+      alert('Please input valid e-mail');
+    } else {
+      // TODO MAKE EMAIL SUBSCRIBE FUNCTION
+    }
+  }
```
Comment (valentine88) [inline app/javascripts/about/emailContainer/emailContainer.jsx]: insert enter

File: app/javascripts/about/emailContainer/emailContainer.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+.emailContainer {
+  width: 100%;
+  height: 338.5px;
+  background-color: #2d2c44;
+  color: white;
+}
+.emailTitle{
```
Comment (valentine88) [inline app/javascripts/about/emailContainer/emailContainer.scss]: insert enter between css element

File: app/javascripts/about/emailContainer/emailContainer.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+}
+.emailInput {
+
+}
```
Comment (valentine88) [inline app/javascripts/about/emailContainer/emailContainer.scss]: same here

File: app/javascripts/root/index.jsx [hunk 4]
```diff
@@ -31,14 +28,12 @@ class Root extends React.PureComponent {
...
         <UpdateBlocker location={location}>
           <Switch>
-            <Route exact path="/" component={About} />
-            <Route path="/about" component={About} />
-            <Route path="/rule" component={Rule} />
-            <Route path="/make_game" component={MakeGame} />
-            <Route path="/play_slot" component={PlaySlot} />
+            <Route exact path='/' component={About} />
+            <Route exact path='/slot' component={Slot} />
+            <Route exact path='/slot/play' component={Slot} />
```
Comment (valentine88) [inline app/javascripts/root/index.jsx]: handle this as `/slot/:page`
and at slot page, handle it with `match` method in React-Router-Dom

File: app/javascripts/slot/index.jsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+import PlaySlot from '../playSlot';
+import Status404 from '../404';
+
+const Slot = () => {
+  return (
+    <div className={styles.slotContainer}>
+      <Header />
+      <h1>Slot Page</h1>
+      <Switch>
+        <Route exact path='/slot' component={PlaySlot} />
```
Comment (valentine88) [inline app/javascripts/slot/index.jsx]: This is what I said before.
it's kind of unnecessary redundancy

File: app/javascripts/slot/layouts/footer.scss [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  color: #F2F2F2;
+  font-size: 18px;
+}
+.item:hover {
+  background: #F2F2F2;
+  color: #6C808C;
+}
+
+.item:active, .item.active {
+  background: #F2F2F2;
```
Comment (valentine88) [inline app/javascripts/slot/layouts/footer.scss]: please use lowercase to color code.

File: app/javascripts/slot/layouts/header.jsx [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import styles from './header.scss';
+
+const Header = () => (
+  <div className={styles.header}>
+    <h1 className={styles.title}> SLOT & SLOT </h1>
+    <NavLink exact to='/slot/play' className={styles.item} activeClassName='active'>PLAY</NavLink>
+    <NavLink exact to='/slot/make' className={styles.item} activeClassName='active'>MAKE</NavLink>
+    <div className={styles.walletStatus}> Your balence : 0.02342 ETH</div>
```
Comment (valentine88) [inline app/javascripts/slot/layouts/header.jsx]: You can connect balance now.
But if you don't know how to do that, I'll handle it.

File: app/javascripts/slot/slot.scss [hunk 8]
```diff
@@ -0,0 +1,3 @@
+.slotComponent{
+  background-color: blue;
```
Comment (valentine88) [inline app/javascripts/slot/slot.scss]: Use color code rather than color name

**Pull request merged**